src/system/database/drivers/mysqli/mysqli_result.php

```diff
@@ -237,4 +237,11 @@ protected function _fetch_object($class_name = 'stdClass')
 		return $this->result_id->fetch_object($class_name);
 	}
 
+	function next_result()
```

Comment:
why this method is needed,
is there any specific reason we declare the method here in the framework ?

Reply:
Without next_result() function, codeigniter does not allow us running multiple procedure after each other,  they always keep the record and block DB mechanism, I don't know why codeigniter like this, below is the reference which is research for the solution yesterday:

https://stackoverflow.com/questions/48256848/codeigniter-3-1-6-commands-out-of-sync-you-cant-run-this-command-now

https://dev.mysql.com/doc/refman/5.7/en/mysql-next-result.html

Reply:
OK, but can you move it to a util or helper ?

Reply:
this one belong to `mysql driver core` of php, we could not move it to another class ( like utils),  It needs some private property just exists inside mysql driver like: `connection`

Reply:
OK